packages/core/src/common/classes.ts

```diff
@@ -24,6 +24,7 @@ export const FIXED = "pt-fixed";
 export const FIXED_TOP = "pt-fixed-top";
 export const VERTICAL = "pt-vertical";
 export const ROUND = "pt-round";
+export const TEXT_OVERFLOW_ELLIPSIS = "pt-text-overflow-ellipsis";
```

Comment:
@palantir/blueprint Should we just put every CSS API class into this file? I'm thinking we might as well.